front/src/modules/trainschedule/components/ManageTrainSchedule/TrainSettings.tsx

```diff
@@ -74,6 +74,7 @@ export default function TrainSettings() {
     setName(nameFromStore);
     setDepartureTime(departureTimeFromStore);
     setInitialSpeed(initialSpeedFromStore);
+    setStartTime(startTimeFromStore.substring(0, 19));
```

Comment:
The startTime stored in the trainschedule is in UTC so we need to convert it. 
Maybe you could create a function `convertUTCToLocalTime()` in `timeManipulation.ts`
`
export function convertIsoUtcToLocalTime(isoDate: string) {
   return dayjs.utc(isoDate).local().format()
}
`

```suggestion
    setStartTime(convertIsoUtcToLocalTime(startTimeFromStore).substring(0, 19));
```

Reply:
Actually we can keep this line like that. But, as the back send us the startTime in UTC, we should convert it to local time when updating the store. So in adjustConfWithTrainToModify, we should do `dispatch(updateStartTime(convertIsoUtcToLocalTime(start_time)));` with the new utils.